### ISSUE TITLE
Allow `DVC.<placeholder>` to be a list

### ISSUE THREAD

**Issue by PythonFZ** [zincware/py-track#40]
The following will raise an error:

```py
@PyTrack(nb_name="Test01.ipynb")
class Stage:
    def __init__(self):
        self.deps = DVC.deps(['dependecy1'])
    
    def __call__(self, deps):
        self.deps.append(deps)
        
    def run(self):
        print(self.deps[0])

    
@PyTrack(nb_name="Test01.ipynb")
class Stage:
    def __init__(self):
        self.deps = DVC.deps()
    
    def __call__(self, deps1, deps2):
        self.deps = [deps1, deps2]
        
    def run(self):
        print(self.deps[0])
```

This works:

```py
@PyTrack(nb_name="Test01.ipynb")
class Stage:
    def __init__(self):
        self.deps1 = DVC.deps()
        self.deps2 = DVC.deps()
    
    def __call__(self, deps1, deps2):
        self.deps1 = deps1
        self.deps2 = deps2
        
    def run(self):
        print(self.deps1)
        print(self.deps2)
```

**Comment by PythonFZ** [comment]
I guess the only way to solve this, is to have some meta information in the json file, where we associate the class attributes with the respective values. I don't see any other way, how we can distinguish lists of arbitrary length to multiple attributes.
This would always write a json file and therefore also make the checks if a json file is beeing generated obsolete.

**Comment by PythonFZ** [comment]
fixed in #31
